chaoscenter/web/src/controllers/KubernetesChaosInfrastructureReferenceField/KubernetesChaosInfrastructureReferenceField.tsx

```diff
@@ -85,8 +120,13 @@ function KubernetesChaosInfrastructureReferenceFieldController({
           setFieldValue('chaosInfrastructure.environmentID', infrastructure.environmentID, false);
         }
       }}
+      activeEnv={activeEnv}
+      setActiveEnv={setActiveEnv}
```

Comment:
if you club env and active env then we don't have to pass these many set state functions to the view